src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Bmi1.PlatformNotSupported.cs

```diff
@@ -30,22 +30,22 @@ public static class Bmi1
         /// unsigned int _bextr_u32 (unsigned int a, unsigned int start, unsigned int len)
         ///   BEXTR r32a, reg/m32, r32b
         /// </summary>
-        public static uint BitFieldExtract(uint value, uint start, uint length) { throw new PlatformNotSupportedException(); }
+        public static uint BitFieldExtract(uint value, byte start, byte length) { throw new PlatformNotSupportedException(); }
```

Comment:
`byte` here and `ushort` below matches the decsription/ranges given for the `bextr` instruction

> Bit 7:0 of the second source operand specifies the starting bit position of bit extraction. A START value exceeding the operand size will not extract any bits from the second source operand. Bit 15:8 of the second source operand specifies the maximum number of bits (LENGTH) beginning at the START position to extract.